src/encoded/static/components/genome_browser.js

```diff
@@ -522,7 +522,22 @@ class GenomeBrowser extends React.Component {
         if (files.length > 0) {
             tracks = this.filesToTracks(newFiles, this.props.label, domain);
```

Comment:
I take it that this PR is part of an addition to a different ticket...?

Reply:
No, this is the only change.